pkgs/applications/networking/libcoap/default.nix

```diff
@@ -1,26 +1,29 @@
-{ fetchFromGitHub, automake, autoconf, which, pkg-config, libtool, lib, stdenv }:
+{ fetchFromGitHub, automake, autoconf, which, pkg-config, libtool, lib, stdenv, gnutls, asciidoc, doxygen
+, tls ? true
+, docs ? true
+}:
 stdenv.mkDerivation rec {
   pname = "libcoap";
-  version = "4.2.1";
+  version = "4.3.0";
   src = fetchFromGitHub {
     repo = "libcoap";
     owner = "obgm";
     rev = "v${version}";
     fetchSubmodules = true;
-    sha256 = "1jkvha52lic13f13hnppizkl80bb2rciayb5hxici0gj6spphgha";
+    sha256 = "1l031ys833gch600g9g3lvbsr4nysx6glbbj4lwvx3ywl0jr6l9k";
   };
   nativeBuildInputs = [
     automake
     autoconf
     which
     libtool
     pkg-config
-  ];
+  ] ++ lib.optional tls gnutls ++ lib.optionals docs [ doxygen asciidoc ] ;
   preConfigure = "./autogen.sh";
   configureFlags = [
     "--disable-documentation"
```

Comment:
Does it still build the documentation, if it finds the relevant software?

Reply:
Good catch! Thank you.